samples/getting-started/azure-quantum/Getting started with Qiskit and Honeywell on Azure Quantum.ipynb

```diff
@@ -0,0 +1,327 @@
+    "## Connecting to the Azure Quantum service\n",
+    "\n",
+    "To connect to the Azure Quantum service, find the resource ID and location of your Workspace from the Azure Quantum portal here: https://portal.azure.com. Navigate to your Azure Quantum workspace and copy the values from the header.\n",
+    "\n",
+    "<img src=\"azure-quantum-resource-id.png\">"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "bb6420f7",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Enter your workspace details here\n",
+    "# Find your resource ID and location via portal.azure.com\n",
+    "provider = AzureQuantumProvider(\n",
+    "  resource_id=\"\",\n",
+    "  location=\"\"\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "id": "f6ce6bbb",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "['ionq.simulator', 'ionq.qpu', 'honeywell.hqs-lt-s1', 'honeywell.hqs-lt-s1-apival', 'honeywell.hqs-lt-s1-sim']\n"
+     ]
+    }
+   ],
+   "source": [
+    "print([backend.name() for backend in provider.backends()])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "44ffcbe3",
+   "metadata": {},
+   "source": [
+    "### Run on API validator (note that this backend will always return 0 on measurement)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "id": "de662326",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Get Honeywell's API validator backend:\n",
+    "apival_backend = provider.get_backend(\"honeywell.hqs-lt-s1-apival\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "id": "9215659e",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/html": [
+       "<pre style=\"word-wrap: normal;white-space: pre;background: #fff0;line-height: 1.1;font-family: &quot;Courier New&quot;,Courier,monospace\">     ┌───┐          ┌─┐      \n",
+       "q_0: ┤ H ├──■───────┤M├──────\n",
+       "     └───┘┌─┴─┐     └╥┘┌─┐   \n",
+       "q_1: ─────┤ X ├──■───╫─┤M├───\n",
+       "          └───┘┌─┴─┐ ║ └╥┘┌─┐\n",
+       "q_2: ──────────┤ X ├─╫──╫─┤M├\n",
+       "               └───┘ ║  ║ └╥┘\n",
+       "c: 3/════════════════╩══╩══╩═\n",
+       "                     0  1  2 </pre>"
+      ],
+      "text/plain": [
+       "     ┌───┐          ┌─┐      \n",
+       "q_0: ┤ H ├──■───────┤M├──────\n",
+       "     └───┘┌─┴─┐     └╥┘┌─┐   \n",
+       "q_1: ─────┤ X ├──■───╫─┤M├───\n",
+       "          └───┘┌─┴─┐ ║ └╥┘┌─┐\n",
+       "q_2: ──────────┤ X ├─╫──╫─┤M├\n",
+       "               └───┘ ║  ║ └╥┘\n",
+       "c: 3/════════════════╩══╩══╩═\n",
+       "                     0  1  2 "
+      ]
+     },
+     "execution_count": 5,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# Create a Quantum Circuit acting on the q register\n",
+    "circuit = QuantumCircuit(3, 3)\n",
```

Comment:
I think here it's also out of scope for this notebook to explain how the 3rd party library works.

Reply:
Agreed in general, but I mentioned that in this case since it's a bit difficult to read the example without knowing that; the calls to `H` and `CX` are a bit easier to read out of context, I guess?

Reply:
I think the notebooks are generally meant for users who are already familiar with Qiskit, so I don't know if this needs to be further clarified.

Reply:
As in the other example, users may discover our samples by any number of different paths. Given that it's not clear reading this code that `circuit` acts on three qubits and three classical bits, I would suggest a comment explaining the code in our sample. Alternatively, if we want to specialize this sample to _require_ Qiskit experience, then we should make that clear in the notebook itself along with links as to further resources.

Reply:
I've added a note that we assume some level of Qiskit familiarity, with a link to their docs. Let me know if this addresses your concern, or if you'd recommend a different approach.